api/envoy/config/filter/http/jwt_authn/v2alpha/config.proto

```diff
@@ -183,33 +183,67 @@ message JwtHeader {
   string value_prefix = 2;
 }
 
-// This is the Envoy HTTP filter config for JWT authentication.
-// [#not-implemented-hide:]
-message JwtAuthentication {
-  // List of JWT rules to valide.
-  repeated JwtRule rules = 1;
+// This message specifies a Jwt requirement.
+//
+// .. code-block:: yaml
+//
+//    required_issuers: [jwt_issuer1, jwt_issuer2]
+//
+// PerRouteConfig for jwt_authn filter uses this message.
+message JwtRequirement {
+  // Specify list of required Jwt issuers. If list is empty,  Jwt is NOT required.
+  repeated string required_issuers = 1;
```

Comment:
Can you add comment to clarify if all are required or just one.

Reply:
All are required.  Updated the comment.